Increase imaging performance (#517)
dcs4cop/xcube: CHANGES.md

  * XCUBE_SH_VERSION=0.9.0.dev0
  * XCUBE_CCI_VERSION=0.9.0.dev0
  * XCUBE_CDS_VERSION=0.9.0

> * Numerous classes and functions became unused in the xcube 0.9 
>   code base and have been removed, also because we believe there is 
>   quite rare outside use, if at all. 

Reviewer: ```suggestion
* Numerous classes and functions became unnecessary in the xcube 0.9 
  code base and have been removed, also because we believe there is 
  quite rare outside use, if any. 
```
Statt unnecessary sonst obsolete?

Author: obsolete!

  
  Removed from `xcube.util.tiledimage`:
  * class `DownsamplingImage`
  * class `PilDownsamplingImage`
  * class `NdarrayDownsamplingImage`
  * function `create_pil_downsampling_image()`
  * function `create_ndarray_downsampling_image()`
  * function `downsample_ndarray()`
  * functions `aggregate_ndarray_xxx()`
  
  Removed from `xcube.util.tilegrid`:
  * functions `pow2_2d_subdivision()`

>   * functions `pow2_1d_subdivision()`

Reviewer: This list is not complete. Should it be?

Author: Let's complete it while we go ahead!
